python/1. introduction/1. python-and-history.mdx

```diff
@@ -1 +1,50 @@
-# Python and It's history by Divyansh
+## Python and It's history
+
+Python was a high level programming language that was originally developed by Guido van Rossum, and first released on February 20,1991
+
+Python's development began in the late 1980s when Guido van Rossum,ā Dutch programmer, started working on it during his Christmas holidays in December 1989.
+
+He was inspired by the ABC language and wanted to create a language that was easy to read and had a clean syntax. 
+
+Python's first official release, Python 0.9.0,occured in February 1991.This version includedexception handling,functions, and modules.
+
+The name "Python" was chosen as a tribute to the British comedy grouo Monty Python. 
+
+
+## Why Python is still relevant
+
+Python remains relevant for a multitude of reasons, which contribute to its continued popularity and widespread use in various fields.
+
+*__Ease of Learning and Readability__:
```

Comment:
Since you've already used __ to make the title bold, please omit the stars at the beginning of each point.